onnxscript/tests/models/signal_dft.py

```diff
@@ -145,7 +136,7 @@ def dft_last_axis(
     where the last axis indicates whether it is the real or the imaginary part.
 
     Args:
-        x: float tensor, the last dimension is the complex one, if has 1
+        x: float tensor, the last dimension is the complex one, if has 1dim1_size
```

Comment:
1dim1_size Is intended or typo?

Reply:
Typo, thanks for catching it. Well, it was mis-directed typing, I was searching for `dim1_size`, I think :-)